scripts/sbom_scraper.sh

```diff
@@ -53,18 +53,20 @@ then
     AUTHOR_EMAIL="$(git config user.email || true)"
     TOOL_NAME="$(git config --get remote.origin.url) $(git ls-files --full-name "$SCRIPTNAME")"
     TOOL_VERSION=$(git describe --tags)${GIT_STATUS:++}
+    SUPPLIER_NAME=github
```

Comment:
Let's remove this please, GitHub is not the SUPPLIER of the component.

Let's remove thus whole git machinery and require that AUTHOR_NAME,  AUTHOR_EMAIL and are provided as options.

And hard code TOOL_NAME to ${SCRIPTNAME}

TOOL_VERSION=$(git describe --tags)